addons/sales_team/models/crm_team.py

```diff
@@ -26,16 +34,213 @@ def _get_default_team_id(self, user_id=None):
                 team_id = default_team_id
         return team_id
 
-    name = fields.Char('Sales Team', required=True, translate=True)
-    active = fields.Boolean(default=True, help="If the active field is set to false, it will allow you to hide the sales team without removing it.")
+    name = fields.Char('Sales Channel', required=True, translate=True)
+    active = fields.Boolean(default=True, help="If the active field is set to false, it will allow you to hide the sales channel without removing it.")
     company_id = fields.Many2one('res.company', string='Company',
                                  default=lambda self: self.env['res.company']._company_default_get('crm.team'))
-    user_id = fields.Many2one('res.users', string='Team Leader')
-    member_ids = fields.One2many('res.users', 'sale_team_id', string='Team Members')
+    currency_id = fields.Many2one(
+        "res.currency", related='company_id.currency_id',
+        string="Currency", readonly=True, required=True)
+    user_id = fields.Many2one('res.users', string='Channel Leader')
+    member_ids = fields.One2many('res.users', 'sale_team_id', string='Channel Members')
     reply_to = fields.Char(string='Reply-To',
-                           help="The email address put in the 'Reply-To' of all emails sent by Odoo about cases in this sales team")
-    color = fields.Integer(string='Color Index', help="The color of the team")
+                           help="The email address put in the 'Reply-To' of all emails sent by Odoo about cases in this sales channel")
+    color = fields.Integer(string='Color Index', help="The color of the channel")
+    team_type = fields.Selection([('sales', 'Sales'), ('website', 'Website')], string='Channel Type', default='sales', required=True,
+                                 help="The type of this channel, it will define the resources this channel uses.")
+    dashboard_button_name = fields.Char(string="Dashboard Button", compute='_compute_dashboard_button_name')
+    kanban_dashboard_graph_data = fields.Text(compute='_compute_kanban_dashboard_graph_data')
```

Comment:
You're probably right, I'll look into it.

(I blindly copied the way our accounting dashboard works, it might need some rework too)